recipes/nodejs/all/conanfile.py

```diff
@@ -1,48 +1,82 @@
 import os
-from conans import ConanFile, tools
-from conans.errors import ConanInvalidConfiguration
+import re
+from conans import __version__
+from conan import ConanFile
+from conan.tools.scm import Version
+from conan.tools.files import copy, get
+from conan.errors import ConanInvalidConfiguration
 
-required_conan_version = ">=1.33.0"
+from six import StringIO
+
+required_conan_version = ">=1.59.0"
 
 
 class NodejsConan(ConanFile):
     name = "nodejs"
     description = "nodejs binaries for use in recipes"
-    topics = ("conan", "node", "nodejs")
+    topics = ("node", "nodejs")
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://nodejs.org"
     license = "MIT"
-    settings = "os", "arch"
+    settings = "os", "arch", "compiler"
     no_copy_source = True
+    short_paths = True
 
     @property
     def _source_subfolder(self):
         return os.path.join(self.source_folder, "source_subfolder")
 
     @property
     def _nodejs_arch(self):
-        if str(self.settings.os) == "Linux":
-            if str(self.settings.arch).startswith("armv7"):
+        if str(self.info.settings.os) == "Linux":
+            if str(self.info.settings.arch).startswith("armv7"):
                 return "armv7"
-            if str(self.settings.arch).startswith("armv8") and "32" not in str(self.settings.arch):
+            if str(self.info.settings.arch).startswith("armv8") and "32" not in str(self.info.settings.arch):
                 return "armv8"
-        return str(self.settings.arch)
+        return str(self.info.settings.arch)
+
+    @property
+    def _glibc_version(self):
```

Comment:
Why is this required?

Reply:
Test will fail otherwise. See supported platforms here: https://github.com/nodejs/node/blob/main/BUILDING.md#platform-list It states glib>=2.17